src/compartment.c

```diff
@@ -399,13 +399,15 @@ comp_map(struct Compartment* to_map)
                       MAP_PRIVATE | MAP_FIXED | MAP_ANONYMOUS,
                       -1, 0);
     assert(map_result != MAP_FAILED);
+    printf("Mapped scratch memory at %p (size %zu).\n", map_result, to_map->scratch_mem_size);
```

Comment:
Are these `printf`s useful? I think they're debugging in disguise?

Reply:
I've done a pass of removing what I could find in https://github.com/capablevms/CHERI-ELF-comp/pull/3/commits/00073d1d86c091d037fc9ba6e931856eda0333fd, and lifting some to code comments.